.bash_nav

```diff
@@ -7,3 +7,28 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>'
+# to remove a shortcut: run 'cds - <shortcut_name>'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
```

Comment:
Now, we check what the first parameter is. If it's a "+", we'll need to create a new shortcut. If it's a "-", we'll need to remove a shortcut.
Otherwise, we'll want to actually jump to a directory.